strawberry/cli/commands/server.py

```diff
@@ -15,8 +14,16 @@
 @click.argument("module", type=str)
 @click.option("-h", "--host", default="0.0.0.0", type=str)
 @click.option("-p", "--port", default=8000, type=int)
-def server(module, host, port):
-    sys.path.append(os.getcwd())
+@click.option(
+    "--app-dir",
+    default=".",
+    type=str,
+    show_default=True,
+    help="Look for the module in the specified directory, by adding this to the "
+    "PYTHONPATH. Defaults to the current working directory.",
```

Comment:
```suggestion
    help=(
        "Look for the module in the specified directory, by adding this to the "
        "PYTHONPATH. Defaults to the current working directory. "
        "Works the same as `--app-dir` in uvicorn."
    ),
```

@BryceBeagle does this sound good?

Reply:
Looks perfect :+1: